Atomic/storage.py

```diff
@@ -0,0 +1,59 @@
+import errno
+import shutil
+import selinux
+import requests
+import subprocess
+import os, sys
+from . import util
+from .Export import export_docker
+from .Import import import_docker
+from .util import NoDockerDaemon
+
+try:
+    from . import Atomic
+except ImportError:
+    from atomic import Atomic
+
+class Storage(Atomic):
+    def reset(self):
+        enc = sys.getdefaultencoding()
+        DriverStorage = {}
+        for i in self.d.info()["DriverStatus"]:
+            DriverStorage[i[0]] = i[1]
+        loopdev = DriverStorage["Data file"]
+        loopfile = DriverStorage["Data loop file"]
+        name = DriverStorage["Pool Name"]
+        root = self.d.info()["DockerRootDir"].decode(enc)
+
+        unit="docker"
+        try:
+            subprocess.check_call(["systemctl", "stop", unit])
+        except:
+            unit = "docker-latest"
+            subprocess.check_call(["systemctl", "stop", unit])
+        
+        if loopfile == "" and loopdev == "":
+            out = subprocess.check_call(["lvmremove", "/dev/mapper/%s" % name.decode(enc)])
```

Comment:
In that case let us first deactivate the volume using "lvchange" and then call "lvremove". --force can imply lot of things and can be dangerous. Want to avoid using --force until and unless we implement a --force option in atomic reset.